study.md

```diff
@@ -43,7 +43,10 @@ Although both Array and Enumerable have a `count` method, Array also defines the
 method `length`.  Why is `length` sensibly defined on Array but not Enumerable?
 
 ```md
-<!-- your answer here -->
+Length on Array is sensibly defined because the length is important for accessing a pair of arguments. Also to keep track of own length at all times to know how many data we are working with - to iterate of the array.
+
+Enumerable have methods which rely on an ordering between members of the collection.
```

Comment:
`length` makes sense on `Array` because arrays have a finite length.  It does not make sense for an `Enumerable` to have a `length` because it might not be finite.  For example, an enumerable that returns a digit of π on each iteration does not have a finite length.